lib/katello/plugin.rb

```diff
@@ -91,89 +91,85 @@
 
     divider :top_menu, :caption => N_('Content Types'), :parent => :content_menu
 
-    if ::Katello::RepositoryTypeManager.enabled?(::Katello::Repository::DEB_TYPE)
-      menu :top_menu,
-           :debs,
-           :caption => N_('Deb Packages'),
-           :url => '/debs',
-           :url_hash => {:controller => 'katello/api/v2/debs',
-                         :action => 'index'},
-           :engine => Katello::Engine,
-           :turbolinks => false
-    end
+    menu :top_menu,
+         :debs,
+         :caption => N_('Deb Packages'),
+         :url => '/debs',
+         :url_hash => {:controller => 'katello/api/v2/debs',
+                       :action => 'index'},
+         :engine => Katello::Engine,
+         :turbolinks => false,
+         :if => lambda { ::Katello::RepositoryTypeManager.enabled?(::Katello::Repository::DEB_TYPE) }
```

Comment:
This `:if` doesn't work in Foreman anymore. It should be fairly straightforward to add it back in: https://projects.theforeman.org/issues/32883